proteus/mprans/RANS2P2D.h

```diff
@@ -334,6 +334,27 @@ namespace proteus
                                    int* csrColumnOffsets_eb_w_w,                                   
                                    int* elementFlags,
 				   int* boundaryFlags)=0;
+    virtual void calculateVelocityAverage(int nExteriorElementBoundaries_global,
```

Comment:
Have you just moved the location of the two-phase operators here?  I think that's a good idea, but I just want to be sure I haven't missed a change to how they're being calculated.  I guess the same question applies for the 3D code above.
I think you've made some changes to the `calculateVelocityAverage` function but I'm not familiar with this function so I don't have anything to add about those changes.

Reply:
Yes, I just did that to make it easier to diff RANS2P.h with the prior version and with RANS2P2D.h.